src/test/java/org/datadog/jmxfetch/TestInstance.java

```diff
@@ -7,13 +7,8 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-import com.google.common.collect.Lists;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
minor: afaik the styleguide says don't use wildcard imports?

Reply:
doesn't apply to test

Reply:
Actually this is up to @truthbk - wildcard imports (and some unused imports) are common in the test folder.

Reply:
Feel free to ignore but @mcculls is right, I know we already have a bunch of wildcard imports in the test package, but let's try to avoid adding more, if possible.

Reply:
ok will address this